src/Microsoft.ML.Core/Data/MetadataUtils.cs

```diff
@@ -335,6 +335,20 @@ public static bool HasKeyNames(this ISchema schema, int col, int keyCount)
                 && type.ItemType.IsText;
         }
 
+        /// <summary>
+        /// Returns whether a column has the <see cref="Kinds.IsNormalized"/> metadata set to true.
```

Comment:
(minor) Do we instead want to document what `IsNormalized` means?  "Returns whether a column `<insert what IsNormalized means>`".

Reply:
Sure, I can add a bit more information. Though that information, I might prefer to make that part of the `Kinds` static class, since the documentation of those are the primary source of truth. This is just something we added as a convenience on top of that.

---
In reply to: [199596084](https://github.com/dotnet/machinelearning/pull/446#discussion_r199596084) [](ancestors = 199596084)